src/sass/extends/_global-header.scss

```diff
@@ -312,10 +313,40 @@ $fa-search: url($image-assets-path + '/font-awesome-svg/search.svg');
     }
   }
 
+  .partner {
```

Comment:
Looks like there is a new option within the global header spec to include a partner logo instead of the ASU logo. Interestingly, this option doesn't appear to exist within the Preact version of the header.

Link: https://unity.web.asu.edu/@asu-design-system/bootstrap4-theme/index.html?path=/story/components-global-header--partner

The inclusion of this option here shouldn't effect anything we have in production. It does represent a change we would need to make in our header script to allow for such an option/configuration to exist.